driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTest.java

```diff
@@ -890,7 +893,7 @@ private List<CommandEvent> lastTwoCommandEvents(final TestCommandListener listen
         return events.subList(events.size() - 2, events.size());
     }
 
-    private void addInitialData() {
+    private BsonDocument addInitialData() {
```

Comment:
Let's rename the method

```suggestion
    private BsonDocument addInitialDataAndGetClusterTime() {
```

Reply:
That name makes me not want to return the cluster time from this method, and instead just get the cluster time separately.  But which do you prefer?

Reply:
I am OK with returning the cluster time from `addInitialData` because it makes the dependency more explicit. If we separate `addInitialData` and `getCurrentClusterTime`, we may need a comment emphasizing the dependency, such that we don't accidentally mess up the order these methods are to be called in the future.

Reply:
Done.

Reply:
(I didn't accept your edit because the caller needs to change too)